pagopa/pn-frontend
# PN-9010 - Refactoring Mixpanel utility functions

## Short description
This PR solves: [PN-9010](https://pagopa.atlassian.net/browse/PN-9010), [PN-9980](https://pagopa.atlassian.net/browse/PN-9980) e risolve i problemi segnalati nel commento di questo task: [PN-10175](https://pagopa.atlassian.net/browse/PN-10175)

## List of changes proposed in this pull request
- Changed structure of tracking function
- Fix some bugs on profile properties and events

## How to test
Start all APPs and check if mixpanel events are called

[PN-9010]: https://pagopa.atlassian.net/browse/PN-9010?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PN-9980]: https://pagopa.atlassian.net/browse/PN-9980?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PN-10175]: https://pagopa.atlassian.net/browse/PN-10175?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
